cypress.config.js

```diff
@@ -42,34 +42,40 @@ const getDevBaseUrl = () => {
   return null;
 };
 
-const e2e = {
-  specPattern: 'cypress/e2e/*.cy.js',
-  testIsolation: false,
-  async setupNodeEvents(on, config) {
-    const webpackOptions = {
-      resolve: {
-        extensions: ['.ts', '.js'],
+async function setupNodeEvents(on, config) {
```

Comment:
I tried to handle this all in a single object, but when imported, the function is not working as expected, so this must be standalone.